authors/apps/authentication/serializers.py

```diff
@@ -30,12 +30,13 @@ class RegistrationSerializer(serializers.ModelSerializer):
 
     # The client should not be able to send a token along with a registration
     # request. Making `token` read-only handles that for us.
+    token = serializers.CharField(max_length=225, read_only=True)
 
     class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password']
+        fields = ['email', 'username', 'password', 'token']
```

Comment:
@MandelaK, I will get back to you later on whether or not we should return tokens on registration.

Reply:
Okay

Reply:
@MandelaK lets take out the token on registration.

Reply:
Okay, working on it